its/tests/src/test/java/its/SonarQubeDeveloperEditionTests.java

```diff
@@ -837,8 +837,13 @@ void shouldSyncTaintVulnerabilities() throws ExecutionException, InterruptedExce
 
       assertThat(taintVulnerability.getType()).isEqualTo(org.sonarsource.sonarlint.core.rpc.protocol.common.RuleType.VULNERABILITY);
       assertThat(taintVulnerability.getRuleDescriptionContextKey()).isEqualTo("java_se");
-      if (ORCHESTRATOR.getServer().version().isGreaterThanOrEquals(10, 2)) {
+      if (ORCHESTRATOR.getServer().version().isGreaterThanOrEquals(10, 8)) {
```

Comment:
Why did you include these changes?

Reply:
The test also broke on the master because of changes coming with SQ 10.8; otherwise, the `branch-10.7` is red but unrelated to my changes.